chart/ohmyglb/values.yaml

```diff
@@ -0,0 +1,65 @@
+global:
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+
+ohmyglb:
+  image: ytsarev/ohmyglb:v0.2.1
+  ingressNamespace: "ohmyglb"
+
+externaldns:
+  image: registry.opensource.zalan.do/teapot/external-dns:latest
+
+etcd-operator:
+  customResources:
+    createEtcdClusterCRD: true
+  etcdOperator:
+    image:
+      repository: quay.io/coreos/etcd-operator
+  backupOperator:
+    image:
+      repository: quay.io/coreos/etcd-operator
+  restoreOperator:
+    image:
+      repository: quay.io/coreos/etcd-operator
+  etcdCluster:
+    image:
+      repository: quay.io/coreos/etcd
+    pod:
+      busyboxImage: busybox:1.28.0-glibc
+
+coredns:
+  image:
+    repository: coredns/coredns
+    tag: "1.6.6"
+  servers:
+  - zones:
+    - zone: .
+    port: 53
+    plugins:
+    - name: cache
+      parameters: 30
+    - name: errors
+    # Serves a /health endpoint on :8080, required for livenessProbe
+    - name: health
+    # Serves a /ready endpoint on :8181, required for readinessProbe
+    - name: ready
+    # Required to query kubernetes API for data
+    - name: kubernetes
+      parameters: cluster.local
+    - name: loadbalance
+      parameters: round_robin
+    # Serves a /metrics endpoint on :9153, required for serviceMonitor
+    - name: prometheus
+      parameters: 0.0.0.0:9153
+    - name: forward
+      parameters: . /etc/resolv.conf
+    - name: etcd
+      parameters: absa.internal
```

Comment:
For future perhaps but refrain from using Absa in any configuration defaults

Reply:
@donovanmuller `example.com` ?

Reply:
Yeah 👍